include/screen.h

```diff
@@ -0,0 +1,39 @@
+// screen.h
+//
+#include <SDL.h>
+#include <string>
+
+#include "color.h"
+
+#ifndef SCREEN_H_
+#define SCREEN_H_
+
+namespace vmonitor {
+class Screen {
+private:
+	SDL_Window* m_window;
+	SDL_Renderer* m_renderer;
+	SDL_Texture* m_texture;
+	Uint32* m_buffer;
```

Comment:
If I do this, then I do not have to take care of deleting these, since smart pointer does that automatically?